### PR TITLE
Updated strings from Crowdin 20220831

### PR DESCRIPTION
**Pull request by BrayanDSO**
Unfornutately, after #12139, so there are new translations + new strings are mixed

### CODE REVIEW

**Comment by BrayanDSO** [comment]
I think that getting warnings on PRs from `i18_sync` isn't very useful

File: AnkiDroid/src/main/res/values-it/02-strings.xml [hunk 1]
```diff
@@ -180,9 +180,9 @@
     <string name="import_error_not_apkg_extension">Il file \"%s\" non ha estensione .apkg o .colpkg</string>
     <string name="import_error_load_imported_database">Le sostituzioni di Anki Database (.anki2) non sono ancora supportate. Consultare il manuale per le istruzioni di sostituzione.</string>
     <string name="import_error_content_provider">Il file selezionato non ha potuto essere importato automaticamente da AnkiDroid. Consulta il manuale utente per importare manualmente i file anki: \n %s</string>
-    <string name="import_error_copy_to_cache">Failed to cache file (possibly out of storage space)</string>
+    <string name="import_error_copy_to_cache">copyFileToCache() fallito (possibilmente una mancanza di spazio di archiviazione)</string>
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/res/values-it/02-strings.xml:183]: nope

File: AnkiDroid/src/main/res/values-it/02-strings.xml [hunk 2]
```diff
@@ -180,9 +180,9 @@
...
     <string name="import_error_multiple_colpkg">Sono stati selezionati più file colpkg.</string>
-    <string name="import_error_colpkg_apkg">Both apkg &amp; colpkg files were selected at once.</string>
+    <string name="import_error_colpkg_apkg">Grey bullet=missing, Green bullet=done</string>
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/res/values-it/02-strings.xml:185]: nope

File: AnkiDroid/src/main/res/values-it/02-strings.xml [hunk 3]
```diff
@@ -379,7 +379,7 @@
     <string name="scoped_storage_migrate">Migra</string>
     <string name="search_card_js_api_no_results">La ricerca non ha restituito nessun risultato</string>
     <!--  JS Addons  -->
-    <string name="not_valid_js_addon">%s is not a valid javascript addon package</string>
+    <string name="not_valid_js_addon">Italiano </string>
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/res/values-it/02-strings.xml:382]: nope

**Review by david-allison — APPROVED** [review]

**Comment by BrayanDSO** [comment]
> Message to maintainers, this PR contains strings changes.
> 
> 1. Before merging this PR, it is best to run the "Sync Translations" GitHub action, then make and merge a PR from the i18n_sync branch to get translations cleaned out.
> 2. Then merge this PR, and immediately do another translation PR so the huge change made by this PR's key changes are all by themselves.
> 
> Read more about updating strings on the wiki,
> 
> * [localization-administration](https://github.com/ankidroid/Anki-Android/wiki/Development-Guide#localization-administration)
> * [download-localized-strings](https://github.com/ankidroid/Anki-Android/wiki/Development-Guide#download-localized-strings)

so this should be the "after merge" message. Perhaps making the "before" and "after" messages different would be more helpful

**Pull request merged**